client/app/lib/styles/ModalForm.scss

```diff
@@ -0,0 +1,28 @@
+// scss-lint:disable ImportantRule
```

Comment:
I think there is no need to disable the important rule here. 

Would suggest to explain the nature of this file, and the source of where we get the css from.